lib/Core/ExecutionState.cpp

```diff
@@ -109,7 +109,7 @@ ExecutionState::ExecutionState(const ExecutionState& state):
     coveredNew(state.coveredNew),
     forkDisabled(state.forkDisabled),
     coveredLines(state.coveredLines),
-    ptreeNode(state.ptreeNode),
+    ptreeNode(nullptr), //ptreeNode shouldn't be shared between 2 states
```

Comment:
If you don't copy just default-initialise the `ptreeNode` to `nullptr` in `ExecutionState.h`.